### DISCUSSION TITLE
TypesScript support

### DISCUSSION THREAD

**Discussion by benj-dobs** [manuelmhtr/countries-and-timezones#35] [Ideas]
Hi!

The DefinitelyTyped module for this package is quite out of date and I'm keen to update it. Would you be open to including the `d.ts` files in this package now, or would you rather they stay in DefinitelyTyped?

In general I think keeping types together with the main package is better, since it helps avoid this kind of problem happen in the first place.

Best,

Benjamin

**Comment by manuelmhtr** [comment]
Hi @benj-dobs,
I also think it would be better to keep them with the package. I'be happy to merge a PR with them.

**Reply by benj-dobs** [reply]
Sweet. I'll get on it.
